Test testOverview (check_metrics.TestMetrics) is flaky

It is rather a new test, that is not always passing. It needs to be robustify.

```
Traceback (most recent call last):
  File "test/verify/check-metrics", line 228, in testOverview
    self.assertLessEqual(last_usage, initial_usage)
AssertionError: 40 not less than or equal to 30
```

Example on ubuntu-1804: https://logs-https-cockpit.apps.ci.centos.org/logs/pull-13368-20200108-090909-6bfb5ed2-cockpit-project-cockpit--ubuntu-1804/log.html#77-2

--- a/test/verify/check-metrics
+++ b/test/verify/check-metrics
@@ -237,10 +237,8 @@ class TestMetrics(MachineCase):
 
         m.execute("kill %d" % mem_hog)
         # should go back to initial_usage; often below, due to paged out stuff
-        time.sleep(8)
-        last_usage = progressValue(2)
-        self.assertLessEqual(last_usage, initial_usage)
-        self.assertGreater(last_usage, 10)
+        wait(lambda: progressValue(2) <= initial_usage)
+        self.assertGreater(progressValue(2), 10)
 
 
 if __name__ == '__main__':

